docs/headless-form/changeset/demo/base-demo.md

```diff
@@ -1,68 +0,0 @@
-```hbs template
-<HeadlessForm
```

Comment:
So these are deleted now because...we want folks to only use this with toucan form?

I can see some value in leaving this in for folks who don't use `ToucanForm`

Reply:
> So these are deleted now because...we want folks to only use this with toucan form?

Yeah! Since Toucan Form is a light wrapper around Headless Form and providing Toucan Core components.

> I can see some value in leaving this in for folks who don't use ToucanForm

I'm not quite sure we have a use case of this, where someone would use Headless Form **and** Toucan Core, but _not_ want to use Toucan Form?  Any thoughts? My worry is leaving it in, people will just try and use Headless Form and Toucan Core directly instead of Toucan Form.  We definitely want people to use Toucan Form when they're integrating these two packages together (imo).

One thing that likely _is_ realistic though is someone wanting to use their _own_ components with Headless Form.  But those docs [live in Headless Form](https://ember-headless-form.pages.dev/docs/usage/custom-controls), not here!

Reply:
Yeah! 🎉